src/database/user.service.js

```diff
@@ -0,0 +1,20 @@
+import { UserModel } from './models/user.js';
+
+export class UserService {
+	/**
+	 * Get or Create a User entry based on its ID
+	 */
+	static async fetchUser(key) {
```

Comment:
Usually this is called `getOrCreate` to indicate it's functionality. Just sayin ^^ you can decide

Reply:
in `guild.service.js` we use `fetchGuild` so I picked `fetchUser` as that would match that standard.

Reply:
I would change that as well to stay consistent. Again, its up to you!